packages/vscode-extension/src/constants.ts

```diff
@@ -10,6 +10,13 @@ export const CONSTANTS = {
     SWITCH_DATABASE_COMMAND: 'neo4j.switchDatabase',
     RUN_CYPHER: 'neo4j.runCypher',
     CYPHER_FILE_FROM_SELECTION: 'neo4j.cypherFileFromSelection',
+    ADD_PARAMETER: 'neo4j.addParameter',
+    CLEAR_PARAMETERS: 'neo4j.clearParameters',
+    INTERNAL: {
+      EVAL_PARAMETER: 'neo4j.internal.evalParam',
+      FORCE_DISCONNECT: 'neo4j.internal.forceDisconnect',
+      FORCE_CONNECT: 'neo4j.internal.forceConnect',
+    },
```

Comment:
I've added a few commands that are internal, therefore they can only be used when having access to the vscode api (i.e. in tests)